src/lib/MediaPreview/MediaPreview.vue

```diff
@@ -56,7 +99,8 @@
 import Loader from '../../components/Loader/Loader'
 import SvgIcon from '../../components/SvgIcon/SvgIcon'
 
-import { isImageFile, isVideoFile, isPdfFile } from '../../utils/media-file'
+import { safe } from '../../utils/dompurify'
```

Comment:
De acordo com o comentário do @Dovyski 

```suggestion
import { sanitize } from '../../utils/dompurify'
```

Reply:
Feito, obrigado